lib/topic.ts

```diff
@@ -3,9 +3,10 @@ import { showElementActions } from './kafka';
 import { navigateToKafkaList } from './navigation';
 
 export const navigateToKafkaTopicsList = async function (page: Page, kafkaName: string) {
-  await expect(page.getByText(kafkaName)).toHaveCount(1);
-  await page.locator('a', { hasText: kafkaName }).click();
-  await expect(await page.locator('button[aria-label="Topics"]')).toHaveCount(1);
+  await expect(page.getByText(`${kafkaName}`)).toHaveCount(1);
+  await page.locator('a', { hasText: `${kafkaName}` }).click();
+  await expect(page.locator('section[aria-label="Metrics view"]')).toBeVisible();
```

Comment:
why do we need this?

Reply:
We were waiting for just the menus to be rendered, so we would then select the topics menu.  This change means that we also wait for the content under the menus to be rendered.  Whilst it's possible to click on the menu before the content is there, the change better reflects user interaction.

Reply:
Shouldn't we click on topics and wait for the rendering page there? I as a user will not wait until some metrics section in the dashboard will be visible if I want to see topics.